### ISSUE TITLE
Support for syncing with 1-click-bom.tsv

### ISSUE THREAD

**Issue by kasbah** [xesscorp/KiField#5]
I have been working on a BOM standard of sorts for use with the 1-click-bom extension and my [kitnic.it](https://kitnic.it) site. 

It's a simple tab-separated format (mostly because this is the clip-board format for spreadsheet programs). [There are some docs and examples here](https://github.com/monostable/1clickBOM#usage). How would you feel about adding support to sync with this format?

This would provide a good way to export a design, use the extension features to do search and complete the fields and then sync it back into KiCad. 

To summarise the biggest difference seems to be tabs  and a .tsv extension vs commas with .csv and components with all matching fields grouped together rather than one per line.

Progress: we currently support one way syncing from 1-click-bom to schematic.

- [x] TSV support
- [x] grouped lines output

**Comment by kasbah** [comment]
Should be good now with https://github.com/xesscorp/KiField/commit/1cc7617cb8ad140624def7275a7a41c894a6760e
